### ISSUE TITLE
Add DarktableUCS

### ISSUE THREAD

**Issue by giannissc** [Ogeon/palette#357]
Perceptual color space are very useful for image processing and it would be nice to have this as part of palette

https://eng.aurelienpierre.com/2022/06/interpolating-hue-angles/

**Comment by Ogeon** [comment]
Interesting! So it's specifically made to work well for image processing?

Also, did you mean to link this? https://eng.aurelienpierre.com/2022/02/color-saturation-control-for-the-21th-century/

**Comment by giannissc** [comment]
Yeah that's the correct link. Basically the darktableUCS is a perceptual color space similar in spirit to CAM16. CAM16 is more suitable for color correction and color adaptation and darktableUCS is more suitable for creative color work (it accounts for the Helmholtz–Kohlrausch effect)

**Comment by Ogeon** [comment]
Ok, yeah, that sounds useful and it's nice that there's an official implementation to look at. 🙂 Thanks for pointing it out!
